client/my-sites/post-share/sharing-preview-modal.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import Gridicon from 'gridicons';
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import SharingPreviewPane from 'blocks/sharing-preview-pane';
+
+const SharingPreviewModal = ( props ) => {
+	const previewProps = pick( props, [ 'message', 'siteId', 'postId' ] );
+	const {
+		isVisible,
+		onClose,
+	} = props;
+
+	return (
+		<Dialog isVisible={ isVisible } additionalClassNames="post-share__sharing-preview-modal" >
+			<header className="post-share__sharing-preview-modal-header">
+				<button onClick={ onClose } className="post-share__sharing-preview-modal-close" >
+					<Gridicon icon="cross" />
+				</button>
+			</header>
+			<SharingPreviewPane { ...previewProps } />
```

Comment:
Copying over notes from the other PR:

This is optional, but if we're concerned with pass through props, perhaps:
```jsx
{ children }
```

So in the caller this looks like:
```jsx
<SharingPreviewModal isVisible={ isVisible } onClose={ onClose }>
    <SharingPreviewPane siteId={ siteId } postId={ postId } message={ message } />
</SharingPreviewModal>
```